.github/workflows/build-py-dists.yml

```diff
@@ -17,11 +17,16 @@ jobs:
                       numpy: "numpy~=1.17"
                     - python-version: 3.9
                       numpy: "numpy~=1.19"
+                    - python-version: "3.10"
+                      numpy: "numpy~=1.21"
 
                 exclude:
                     - os: macos-latest
                       python-arch: x86
 
+                    - python-version: "3.10"
+                      python-arch: x86

```

Comment:
Does python 3.10 not support `x86`? I skimmed the changelog and didn't find anything relevant. Maybe worth a comment here explaining why it's excluded.

Reply:
Since invest no longer supports `x86`, could we drop it here as well? Would be nice to halve the number of matrix combinations that have to run.

Reply:
>Since invest no longer supports x86, could we drop it here as well? Would be nice to halve the number of matrix combinations that have to run.

You know, that's an interesting point, and it makes me wonder about the PyPI download stats of the x86 and x64 windows wheels.  I'm not at all against dropping them, but I think for the moment I'd like to keep the x86 builds until we've had a chance to take a closer look, so I've created https://github.com/natcap/pygeoprocessing/issues/232 for this so we can think it through.  How does that sound?

Reply:
Sounds good!